pkg/obfuscator/obfuscator.go

```diff
@@ -6,6 +6,8 @@ type Obfuscator interface {
 	Path(string) string
 	// Contents takes string as input and return the obfuscated string
 	Contents(string) string
+	// Type returns an obfuscator type which would come into help during the reporting.
+	Type() string
```

Comment:
did you actually make use of this?

Reply:
haven't found a reference, so let's remove?

Reply:
This is getting used in `"UpdateReportPerObfuscator"` function where the report for each obfuscate type would be accordingly appended